efficientdet/hparams_config.py

```diff
@@ -275,7 +223,7 @@ def default_detection_configs():
             backbone_name='efficientnet-b1',
             image_size=640,
             fpn_num_filters=88,
-            fpn_cell_repeats=4,
+            fpn_cell_repeats=5,
```

Comment:
Each original BiFPN layer has 2 paths. With fpn_cell_repeats=4, it have 2 * 4 =8 paths in total.
Here you change each layer to have 4 paths. With fpn_cell_repeats=5, you will have 4*5=20 paths in total. 
Is my understanding correct?

Reply:
Yes absolutely right ! It's scales very differently and not as good as EfficientDets but is able to extract more features and hence better accuracy. I tried scaling BiFPN (increases fpn cell repeats) to match the same number of paths in QuadFPN but couldn't increases performance more than 0.1 - 0.2% AP than one reported in your paper , so thought maybe try this out further.